addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -1411,12 +1411,120 @@ QUnit.module("Fields", (hooks) => {
+                    view_in_kanban: true,
+                },
+                {
+                    name: "property_float",
+                    string: "My Float",
+                    type: "float",
+                    value: 12.2,
+                    view_in_kanban: true,
+                },
+                {
+                    name: "property_date",
+                    string: "My Date",
+                    type: "date",
+                    value: "2023-06-05",
+                    view_in_kanban: true,
+                },
+                {
+                    name: "property_datetime",
+                    string: "My Datetime",
+                    type: "datetime",
+                    value: "2023-06-05 11:05:00",
+                    view_in_kanban: true,
+                },
+                {
+                    name: "property_checkbox",
+                    string: "My Checkbox",
+                    type: "boolean",
+                    value: true,
+                    view_in_kanban: true,
+                },
+            ],
+            company_id: 37,
+        });
+
+        await makeView({
+            type: "kanban",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <kanban>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div>
+                                <field name="company_id"/> <hr/>
+                                <field name="display_name"/> <hr/>
+                                <field name="properties" widget="properties"/>
+                            </div>
+                        </t>
+                    </templates>
+                </kanban>`,
+        });
+
+        // check for label in integer, float, date and datetime field
+        assert.strictEqual(
+            target.querySelector(
+                ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(1) label"
+            ).innerText,
+            "My Integer"
+        );
+        assert.strictEqual(
+            target.querySelector(
+                ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(2) label"
+            ).innerText,
+            "My Float"
+        );
+        assert.strictEqual(
+            target.querySelector(
+                ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(3) label"
+            ).innerText,
+            "My Date"
+        );
+        assert.strictEqual(
+            target.querySelector(
+                ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(4) label"
+            ).innerText,
+            "My Datetime"
+        );
+
+        //check that label and border class is present for checkbox field
+        assert.containsOnce(
+            target.querySelector(
+                ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(5)"
+            ),
+            ".border"
+        );
```

Comment:
Can't you simplify this to:

`assert.containsOnce(target, ".o_kanban_record:nth-child(5) .o_kanban_property_field:nth-child(5) .border");`

?